Dark.json

```diff
@@ -33575,7 +33575,7 @@
         "errataNotes": "Power increased from 2 to 3. Previous text: May add 1 pilot. Kylo deploys -3 aboard. While Kylo piloting, immune to attrition < 5 and, unless Leia here, once per battle may lose immunity to attrition to 'spin': add maneuver to power.",
         "errataSymbol": "E1",
         "forfeit": "4",
-        "gametext": "May add 1 pilot. Kylo deploys -2 aboard. May reveal from hand to ▲ Kylo and deploy both simultaneously. While Kylo piloting, Kylo and this TIE are immune to It Can Wait, Rebel Barrier, and attrition < 5.",
+        "gametext": "May add 1 pilot. Kylo deploys -2 aboard. May reveal from hand to take into hand Kylo and deploy both simultaneously. While Kylo piloting, Kylo and this TIE are immune to It Can Wait, Rebel Barrier, and attrition < 5.",
```

Comment:
```suggestion
        "gametext": "May add 1 pilot. Kylo deploys -2 aboard. May reveal from hand to take Kylo into hand from Reserve Deck (reshuffle) and deploy both simultaneously. While Kylo piloting, Kylo and this TIE are immune to It Can Wait, Rebel Barrier, and attrition < 5.",
```

Reply:
Fixed.